tests/unit/schema/test_column_schemas.py

```diff
@@ -180,8 +180,8 @@ def test_list_column_attributes():
     col2_schema = ColumnSchema("col2", is_list=True)
 
     assert col2_schema.is_list
-    assert col2_schema.is_ragged
-    assert col2_schema.quantity == ColumnQuantity.RAGGED_LIST
+    assert not col2_schema.is_ragged
+    assert col2_schema.quantity == ColumnQuantity.FIXED_LIST
```

Comment:
This change reverts the default `is_ragged` value to `False` unless specified with the parameter or with a value count that has max > min.  

This was the case in merlin-core 0.5.0, but changed in 0.6.0 by #118

Reply:
Changed this back to set the default value of is_ragged=True if is_list=True unless the value_count max > min